components/Form.js

```diff
@@ -48,7 +54,8 @@ export default function Form({ onCalculatePrice, value }) {
           name="expenses"
           value={num}
           onChange={handleNumChange}
-          type="number"
+          type="decimal"
```

Comment:
I don´t know about an input of type `decimal`.
You can add the step attribute to make it appear as a decimal field:
–> https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#allowing_decimal_values

Reply:
Yes, but I don't want any decimal places, is it possible with this type number and step?